stable/traefik/README.md

```diff
@@ -87,12 +87,14 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `acme.persistence.size`         | Minimum size of the volume requested                                 | `1Gi`                                     |
 | `dashboard.enabled`             | Whether to enable the Traefik dashboard                              | `false`                                   |
 | `dashboard.domain`              | Domain for the Traefik dashboard                                     | `traefik.example.com`                     |
-| `gzip.enabled`                  | Whether to use gzip compression                                      | `true`                     |
+| `dashboard.basicAuth`           | Basic auth for the Traefik dashboard, see Authentication section     | No auth                                   |
```

Comment:
To allow for the possibility of other auth mechanisms being configurable in the future, I might suggest `dasboard.auth.basic` instead.